wordpress-mobile/WordPress-Android
# Use different style for CommentNotificationsBottomSheetFragment

This PR fixes the Follow Conversation bottom sheet opening animation issue in Post Details.


To test:
- Enable comment snippet FF.
- Navigate to post details, and press on Follow Conversation button, and on the Bell icon afterward.
- Observe that the status bar is not animating with a bottom sheet.
- Repeat this in Dark mode to confirm that everything looks ok.

## Regression Notes
1. Potential unintended areas of impact
- Should be non, since changes are visual.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing.

3. What automated tests I added (or what prevented me from doing so)
Changes are purely visual. 

PR submission checklist:
